Makefile

```diff
@@ -3,29 +3,19 @@
 
 # Use docs for compatibility with GitHub pages.
 out=out
-intermediate=build
 
 all: $(out)/index.html #$(out)/ktx.pdf
 
 inlined_images := icons/masters/ktx_document.svg \
                   icons/win/ktx_document.ico \
                   images/khronos.svg
 
-$(intermediate)/docinfo.html: docinfo.html $(intermediate) inlineimages.pl $(inlined_images)
-	./inlineimages.pl < $< > $@
-
-$(out)/index.html: ktxspec.adoc $(out) $(intermediate)/docinfo.html
-	asciidoctor -a docinfodir=$(intermediate) -D $(dir $@) -o $(notdir $@) $<
+$(out)/index.html: ktxspec.adoc $(out) $(inlined_images) docinfo.html
+	asciidoctor -r ./inline-images.rb -D $(dir $@) -o $(notdir $@) $<
 
 $(out)/ktx.pdf:
 
-$(intermediate) $(out):
-	mkdir -p $@
-
 clean:
-	rm -rf $(intermediate)
-
-clobber: clean
```

Comment:
Still need both clean: & clobber: targets. clean will do nothing. clobber should rm $(out).